lib/interceptor.js

```diff
@@ -1,12 +1,17 @@
 'use strict';
 
 var interceptor = {
-  setup: function setup(done) {
+  disableInterceptor: function disableInterceptor(done) {
+    var NAMESPACE = '__webdriverajax';
+    window[NAMESPACE].interceptorDisabled=true;
```

Comment:
This will throw an error if called before the interceptor is set up--for example, if a page reload has occurred for some reason outside of the test's control.